### ISSUE TITLE
Package tar@3.0.0 (latest) broke unpack.js

### ISSUE THREAD

**Issue by ZaDarkSide** [moos/wordnet-db#9]
```
node_modules\wndb-with-exceptions\unpack.js:52
    .pipe(tar.Extract({ path: __dirname }))
              ^

TypeError: tar.Extract is not a function
    at extractTarball (F:\Repositories\iris-bot-engine.git\node_modules\wndb-with-exceptions\unpack.js:52:15)
    at Object.<anonymous> (F:\Repositories\iris-bot-engine.git\node_modules\wndb-with-exceptions\unpack.js:58:1)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:420:7)
    at startup (bootstrap_node.js:139:9)
```

`tar@2.2.1` was the last version compatible with [unpack.js](https://github.com/moos/wordnet-db/blob/master/unpack.js#L22)

In `tar@3.0.0` method `tar.Extract` was renamed `tar.x` see:
`tar@2.2.1` => [tar.js](https://github.com/npm/node-tar/blob/v2.2.1/tar.js#L173) vs `tar@3.0.0` => [tar.js](https://github.com/npm/node-tar/blob/v3.0.0/tar.js#L12) 

You are using `tar@latest` in [package.json](https://github.com/moos/wordnet-db/blob/master/package.json#L13) it should be `tar@2.2.1`

**Comment by Samurais** [comment]
+1

**Comment by ZaDarkSide** [comment]
Updated [original post](https://github.com/moos/wordnet-db/issues/9#issue-227552815) with more info.

**Comment by laalex** [comment]
+1

**Comment by jscMR** [comment]
+1

**Comment by Cabub** [comment]
+1

**Comment by ZaDarkSide** [comment]
Please accept the pull request from @laalex to fix this issue.

**Comment by moos** [comment]
Should be fixed in 3.1.3 -- publishing shortly.  Sorry for delay.
